app/views/pages/_outside_links.html.haml

```diff
@@ -0,0 +1,21 @@
+%nav.fa-container
+  %a{:href => "https://github.com/444c43"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace